meinberlin/apps/dashboard2/contents.py

```diff
@@ -1,5 +1,27 @@
 class DashboardContents:
-    _registry = {}
+    _registry = {'project': {}, 'module': {}}
+
+    def __getitem__(self, identifier):
+        component = self._registry['project'].get(identifier, None)
+        if not component:
+            component = self._registry['module'].get(identifier)
+        return component
+
+    def __contains__(self, identifier):
+        return (identifier in self._registry['project'] or
+                identifier in self._registry['module'])
```

Comment:
i wasn't sure either, but it can be so convenient to use this method as identifiers are defined to be unique anyway. it would also be possible to store the _registry as a dict of {identifier: ('project', component} 
then the cost of get_project_components would be higher but the general registry would be consistent